apps/web/src/pages/templates/editor_v2/CloudWorkflowSettingsSidePanel.tsx

```diff
@@ -25,7 +27,10 @@ export const CloudWorkflowSettingsSidePanel: FC<CloudWorkflowSettingsSidePanelPr
   return (
     <Sidebar customHeader={<Title variant="section">Workflow settings</Title>} isOpened onClose={onClose}>
       <div className={css({ colorPalette: 'mode.local' })}>
-        <WorkflowSettingsSidePanelContent isLoading={isLoading} />
+        <WorkflowSettingsSidePanelContent
+          isLoading={isLoading}
+          workflowType={searchParams.get('type') as WorkflowTypeEnum}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏Do they all have a type in the url? Could there be a situation where a regular one has no type/`?type=undefined`/ user got a url without the type param or deleted that param

Reply:
They do -- it appears to be how we decide which version of the Workflow Editor to show